src/extra/flash/setup

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+repo="ocawesome101/ulos-external"
+name="external"
+path="/tmp/$name"
+subdir="flash"
+
+printf "downloading source\n"
+if ! [ -e $path ]; then
+  git clone https://github.com/$repo $path
+else
+  oldpwd=$PWD
+  cd $path
+  git pull
+  cd $oldpwd
+fi
+
+rm -rf files && mkdir -p files
+cp -rv /tmp/external/$subdir/* files/
```

Comment:
this file should pull a file from a git repository - i think modifying `setup.example` would work better for this.